internal/daemon/vendor-specific-plugins/marvell/main.go

```diff
@@ -363,14 +531,12 @@ func NewMarvellVspServer(opts ...func(*mrvlVspServer)) *mrvlVspServer {
 }
 
 func main() {
-	// mrvlVspServer := commchannel.NewMarvellVspServer()
 	mrvlVspServer := NewMarvellVspServer()
-	listener, err := mrvlVspServer.Listen()
+	listener, err := mrvlVspServer.Listen(mrvlVspServer)
```

Comment:
Minor complained:

I know I suggested this, but the `mrvlVspServer` argument is unnecessary here.

note that `Listen()` already has:
```
func (vsp *mrvlVspServer) Listen(mrvlVspServerInstance *mrvlVspServer) (net.Listener, error)
```

We don't need the variable `mrvlVspServer`. We already have the `vsp` argument, the *self* pointer for the method call.

Reply:
Talked about during the meeting that the input arg should be removed.